src/Symfony/Component/Validator/ConstraintValidatorFactory.php

```diff
@@ -26,13 +24,6 @@ class ConstraintValidatorFactory implements ConstraintValidatorFactoryInterface
 {
     protected $validators = array();
 
-    private $propertyAccessor;
-
-    public function __construct($propertyAccessor = null)
```

Comment:
this is a BC break:

- it does not allow reusing the same property access for each expression validator anymore for people using the component.
- it breaks code for people extending the class and calling ``parent::__construct``